merlin/systems/dag/ops/softmax_sampling.py

```diff
@@ -99,7 +99,7 @@ def compute_output_schema(
         self, input_schema: Schema, col_selector: ColumnSelector, prev_output_schema: Schema = None
     ) -> Schema:
         """Describe the operator's outputs"""
-        return Schema([ColumnSchema("ordered_ids", dtype=np.int32, is_list=True, is_ragged=False)])
+        return Schema([ColumnSchema("ordered_ids", dtype=np.int32, dims=(None, 1))])
```

Comment:
We now need to specify the second dimensiion output size as 1 so that the output is fixed size and returns a single `ordered_ids` in the Triton config instead of split into values/lengths

Reply:
`(None, 1)` matches the return shape of `(-1, 1)` from the transform method. There may be some follow-up work to make this operator work on more than one batch at a time instead of only the one that it assumes currently.

Reply:
Good catch! Agreed on the follow-up work, that’s been on our list for a while but we decided it would be easier to tackle after we did the shapes migration.